pkg/cli/image/manifest/manifest.go

```diff
@@ -158,13 +158,12 @@ func (o *FilterOptions) Validate() error {
 }
 
 // Complete performs defaulting by OS.
-func (o *FilterOptions) Complete(flags *pflag.FlagSet) error {
+func (o *FilterOptions) Complete(flags *pflag.FlagSet) {
```

Comment:
The rule of thumb is that each of the 3 methods (`Complete`, `Validate` and `Run`) will return error. Please don't change that contract.

Reply:
ok, updated